framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -367,7 +402,14 @@ public int hashCode() {
         return result;
     }
 
-    /** Returns the conditional postconditions on the method {@code methodElement}. */
+    /// Conditional postcondition methods (keep in sync with other two types)
+
+    /**
+     * Returns the conditional postconditions on {@code methodElement}.
+     *
+     * @param methodElement the method or contract whose unconditional postconditions to return
```

Comment:
A constructor cannot have a conditional postcondition. So here `methodElement` should really only be for methods, so the name is appropriate.